services/web/project/service/graph_service.py

```diff
@@ -0,0 +1,26 @@
+import networkx as nx
+import matplotlib.pyplot as plt
+from ..repository import metrics_repository, github_user_repository
+
+
+def load_graph(login):
+    main_user = github_user_repository.get_by_login(login)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [local variable 'main_user' is assigned to but never used (F841)](https://app.codacy.com/manual/mpgabriel95/TCC/pullRequest?prid=5565580)